### PR TITLE
Nmap: relocate to sysutils/ and update to 7.12

### PR DESCRIPTION
**Pull request by jimklimov**

### CODE REVIEW

File: components/sysutils/nmap/Makefile [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+include $(WS_MAKE_RULES)/prep.mk
+include $(WS_MAKE_RULES)/configure.mk
+include $(WS_MAKE_RULES)/ips.mk
+
+PKG_OPTIONS +=	-D COMPONENT_DESCRIPTION="$(COMPONENT_DESCRIPTION)"
+PKG_OPTIONS +=	-D PYTHON_VERSION="$(PYTHON_VERSION)"
+PKG_MACROS +=	PYVER=$(PYTHON_VERSION)
+
+# Can't use the Studio compilers as they don't support the GNU
```
Comment (pyhalov) [inline components/sysutils/nmap/Makefile]: Please, remove next 3 lines

File: components/sysutils/nmap/nmap.p5m [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Portions Copyright (c) 2011, 2013, Oracle and/or its affiliates. All rights reserved.
+# Copyright 2016 Jim Klimov
+#
+
+<transform file path=usr.*/man/.+ -> default mangler.man.stability volatile>
```
Comment (pyhalov) [inline components/sysutils/nmap/nmap.p5m]: why do we need these transforms if we don't deliver contents?

File: components/sysutils/nmap/nmap.p5m [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+set name=pkg.fmri value=pkg:/$(COMPONENT_FMRI)@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.summary value="$(COMPONENT_SUMMARY)"
+set name=info.description value="$(COMPONENT_DESCRIPTION) umbrella for different components"
+set name=info.classification value="$(COMPONENT_CLASSIFICATION)"
+set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
+set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+
+# Legacy data from OpenSolaris recipe
```
Comment (pyhalov) [inline components/sysutils/nmap/nmap.p5m]: we don't need this

**Comment by jimklimov** [comment]
Addressed the PR proposals.

**Comment by xen0l** [comment]
Can you squash the commits?

**Comment by jimklimov** [comment]
Yes, although some other day, and if content is good for you so i don't
have to suffer iterations of forced-pushed across my workspaces. Then
again, you can also click "sqhash and merge" in the PR interface ;)

## 

Typos courtesy of GMail on my Samsung Android
11.06.2016 1:11 пользователь "Adam Števko" notifications@github.com
написал:

Can you squash the commits?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
https://github.com/OpenIndiana/oi-userland/pull/2064#issuecomment-225317027,
or mute the thread
https://github.com/notifications/unsubscribe/ABj5lJSbo2UcxfcVWRVpOvY-v-i_anUDks5qKe8bgaJpZM4IpHpr
.

**Pull request merged**